configure

```diff
@@ -76,17 +76,9 @@ Usage: $0 [OPTION]... [VAR=VALUE]...
                                   - INFO
                                   - DEBUG
                                   - TRACE
-    --with-address-sanitizer    build with address sanitizer if available
-    --with-asan                 alias for --with-address-sanitier
-    --enable-asan               alias for --with-address-sanitier
-    --with-gcov                 build with gcov coverage enabled
+    --with-sanitizers=LIST      build with this list of sanitizers enabled
```

Comment:
I think it would be nice  to have a convenience options such as `--enable-sanitizers` that simply sets whatever we usually use or alternatively list the possible options for `LIST` in the description.

Reply:
I'm not a fan of introducing too many ways for the same thing. I'll give you `--dev-mode` back, ok? 😇